app/src/main/res/layout/module_list.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
it's always better to use more simple layout instead of more complex
e.g. in this case (and in some another layouts) you can replace LinearLayout with FrameLayout
It improves performance and measuring time that has positive effect on UX and battery life